app/src/main/res/layout/list_item_overdue_patient.xml

```diff
@@ -23,6 +23,18 @@
       app:layout_constraintTop_toTopOf="parent"
       tools:src="@drawable/ic_patient_female" />
 
+    <com.google.android.material.checkbox.MaterialCheckBox
+      android:id="@+id/checkbox"
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:layout_marginEnd="@dimen/spacing_12"
+      android:minWidth="@dimen/spacing_0"
+      android:minHeight="@dimen/spacing_0"
```

Comment:
Why are we setting the `minWidth` and `minHeight` here?  I don't think we need it?

Reply:
Because there is some extra padding by default in the checkbox for at the end sending a scree shot in dm